docs/technical_reference/unit_models/ion_exchange_0D.rst

```diff
@@ -80,7 +83,7 @@ The model provides three ports (Pyomo notation in parenthesis):
 
 * Inlet port (inlet)
 * Outlet port (outlet)
-* Regeneration port (regen)
+* Regeneration port (regen, only if not using ``single_use`` resin configuration)
```

Comment:
Its slightly confusing from a mass balance perspective. If set to `single_use`, `inlet == outlet` so how is mass removed from a modeling perspective? There is no spent resin stream that would then be regenerated with x efficiency and then added as the regen stream, correct?

Reply:
Hm maybe a poor choice of words on my part. The mass balance is handled the same as before where effluent concentration is estimated via trapezoid rule, which is then used to set ``process_flow.mass_transfer_term`` (see e.g., ``eq_mass_transfer_target_fr`` - I think this is pretty much identical to how GAC does it) . Without single-use, that flows to the ``regeneration_stream`` but with single-use it goes nowhere. There was never any regeneration efficiency in the model, so anything that flowed to that port was assumed to be removed via regeneration (which is not handled by the model; a user could attach a separate unit process as a regen handling process if they wanted.

But maybe there is virtue in having the removed mass flow somewhere rather than a black hole.

Reply:
I remember this 'into the void' method when originally developing the model. 

I think I'm still confused that if the contaminants are going 'into the void' and the resin is also not modeled with a pseudo-species method, what is the intention of the `regeneration_stream`? i.e., if someone wanted to customize a regen or waste handling unit, what is this stream/port accomplishing?

Reply:
Following up- I added the ``regeneration_stream`` into the model for all CONFIGs (i.e. undoing what I had initially implemented). I also added back backwashing/rinsing costs/params/expressions for the ``single_use`` option. Now only regenerant costs are excluded with the ``single_use`` configuration.